package.json

```diff
@@ -44,21 +45,21 @@
     "@babel/preset-env": "^7.17.10",
     "@babel/preset-react": "^7.16.7",
     "@babel/preset-typescript": "^7.16.7",
+    "@playkit-js/kaltura-player-js": "canary",
```

Comment:
"canary" will be resolved only once, the next yarn install won't update the version. that's why I prefer using a fixed version

Reply:
fixed to latest

Reply:
i have to leave it canary for the build which depends on changes on kaltura player master